.github/workflows/core.yml

```diff
@@ -226,7 +226,7 @@ jobs:
         with:
           submodules: recursive
       - uses: ./.github/actions/environment
-      - run: nix-shell --run "poetry run make -C core build_unix_frozen"
```

Comment:
`make build_unix_frozen` is not setting PYOPT=0, so `cargo clippy` fails due to missing debug-related QSTRs.